src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,71 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[],
+  selectedTodoId: number,
+  onTodoSelection: (id: number) => void,
+}
+export const TodoItem: React.FC<Props> = ({
+  todos,
+  selectedTodoId,
+  onTodoSelection,
+}) => (
+  <tbody>
+    {todos.map(({ id, title, completed }) => {
```

Comment:
why you do it in the todoItem?
In `TodoItem` you should just handle only 1 todo
move it to the `TodoList`